### DISCUSSION TITLE
Streaming/Iterative JSON deserialization

### DISCUSSION THREAD

**Discussion by ariwbolton** [pydantic/pydantic#8205] [Question]
I'm trying to "iteratively deserialize" some JSON into python objects (like [ijson](https://pypi.org/project/ijson/) or [json-streamer](https://github.com/kashifrazzaqui/json-streamer); assume I can't use these for the moment) -- is there any way to do that with pydantic? Specifically, I need to register some kind of event hook for listening to deserialization events whenever a field is updated, as portions of the JSON are parsed, _before_ the entire JSON bytes are available. (I noticed that [jiter](https://github.com/pydantic/jiter) was just added to core; is that functionality exposed to end users somehow?)

**Comment by failable** [comment]
Same question
